bridge/node/src/api.rs

```diff
@@ -0,0 +1,65 @@
+use crate::helper::*;
+use dsnp_graph_config::Config;
+use neon::prelude::*;
+
+// Collection of GraphStates
+//#[allow(clippy::vec_box)]
+//static GRAPH_STATES: Mutex<Vec<Box<GraphState>>> = Mutex::new(Vec::new());
+
+/// Neon implementation of print_hello_graph function
+pub fn print_hello_graph(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+	println!("Hello, Graph!");
+	Ok(cx.undefined())
+}
+
+/// Get graph config from the environment
+/// # Arguments
+/// * `cx` - Neon FunctionContext
+/// * `env` - Neon Environment object
+/// # Returns
+/// * `JsResult<JsObject>` - Neon JsObject containing the graph config
+pub fn get_graph_config(mut cx: FunctionContext) -> JsResult<JsObject> {
+	let environment_obj = cx.argument::<JsObject>(0)?;
+
+	let environment = unsafe { environment_from_js(&mut cx, environment_obj) };
+	let config: &Config = environment.get_config();
```

Comment:
These might need that catch_unwind also but not require at this point of development